packages/manager/modules/vps/src/veeam/modal/schedule-backup/vps-schedule-backup.controller.js

```diff
@@ -0,0 +1,78 @@
+const RESCHEDULE_AUTOMATED_BACKUP_NAMESPACE =
```

Comment:
maybe move this to constants file

Reply:
From my side i populate a constants file with const onlt when i need to share them between files.
In my case i let it into ctrl because i used it only into it.

Reply:
As we discuss i let it into ctrl.
Thanks